README.md

```diff
@@ -11,4 +11,36 @@ Supported formats:
    ```
     gradlew clean build
    ``` 
-3. Find the distribution in build/distributions.
\ No newline at end of file
+3. Find the distribution in build/distributions.
+
+## Transform Parasoft Reports
+
+### Transform Parasoft Static Analysis XML Report to SARIF Report
+1. Run XMLToSARIF script with the generated XML report:
+   * For Windows:
+      ```cmd
+      path\to\XMLToSARIF.bat -i <path\to\Parasoft\XML\report> [-o <path\to\output\SARIF\report>] [-t <path\to\parasoft\tool\installation\or\Java\home\directory>]
+      ```
+   * For Linux:
+      ```shell
+      path/to/XMLToSARIF.sh -i <path/to/Parasoft/XML/report> [-o <path/to/output/SARIF/report>] [-t <path/to/parasoft/tool/installation/or/Java/home/directory>]
+      ```
+2. If a path is provided to the `-o` option, the SARIF report should be generated in the specified path. If not, the SARIF report will be generated in the same directory as input XML report. 
+
+If the feature of checked files list is enabled, the checked files can be found in `artifacts` list in the last part of SARIF report. Also, you can search `artifacts` words directly to find 
```

Comment:
We should move this explanation to the Features section.
We should provide something like a JSONPath instead of telling users to search for it themselves. Example: $.runs[*].artifacts[*].
The key point is to tell users which part or layer the list is in and what it will look like.